src/simpleTrianglePatterns/simpleTrianglePattern2.java

```diff
@@ -8,19 +8,19 @@ public static void main(String[] args){
         System.out.print("Enter a number:");
         try{
             int number = userInputScanner.nextInt();
-        } catch (WrongUserInputException wuie) {
-            System.out.println("Wrong format! Please use numbers to enter the input!");
-        } catch (RunTimeException rte){
-            System.out.println(rte);
-        }
-        for(int i = 1; i <= number; i++){
-            for(int j = i; j <= number; j++){
-                System.out.print(" ");
-            }
-            for(int j = 1;j <= i; j++){
-                System.out.print("*");
+            for(int i = 1; i <= number; i++){
+                for(int j = i; j <= number; j++){
+                    System.out.print(" ");
+                }
+                for(int j = 1;j <= i; j++){
+                    System.out.print("*");
+                }
+                System.out.println();
             }
-            System.out.println();
+        } catch (RuntimeException ex2){
```

Comment:
meaningful names? ex2?? ex1???? there's a case which obvious make the program crash, need to catch the exception! The IDE might be display it.